app/components/common/candidate/candidate_page.html

```diff
@@ -39,6 +39,13 @@
       <div class="component__heading" ng-repeat-start="(label, amount) in $ctrl.contributions_by_type_percentages">{{ label }}</div>
       <odca-money-bar-chart value="{{ amount }}" max="1000" precision="1" format="percentage" color="green" ng-repeat-end></odca-money-bar-chart>
     </div>
+    <div class="see-more" ng-if="!!$ctrl.supporting.total_contributions">
```

Comment:
This is kind of silly, but do you mind changing this to `$ctrl.supporting.total_contributions > 0`? I've seen a bug on the backend that when there's no data, `total_contributions` ends up as a list, and `[] > 0` evaluates true :(